sdk/ml/azure-ai-ml/azure/ai/ml/entities/_feature_store_entity/feature_store_entity.py

```diff
@@ -33,6 +32,7 @@ def __init__(
         name: str,
         version: str,
         index_columns: List[DataColumn],
+        stage: Optional[str] = None,
```

Comment:
Can you add doc string for this parameter?

Reply:
Added for feature set and feature store entity.